src/aks-preview/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM microsoft/azure-cli:latest
```

Comment:
azure-cli docker is migrated to [MCR](https://hub.docker.com/_/microsoft-azure-cli), please use `mcr.microsoft.com/azure-cli:latest`

Reply:
good catch. I actually dont need that file. :)